src/applications/edu-benefits/0994/components/PaymentView.jsx

```diff
@@ -1,8 +1,13 @@
 import _ from 'lodash';
 import React from 'react';
 import { connect } from 'react-redux';
-import { maskBankInformation, hasNewBankInformation } from '../utils';
 import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
```

Comment:
Can we update to the new `va-alert` web component: https://design.va.gov/storybook/?path=/docs/components-va-alert--default

Reply:
It probably should be updated, but maybe as a separate smaller issue instead of piggybacking off of this task. (Not really related)